runtime/util/hshelp.c

```diff
@@ -482,6 +485,185 @@ areSingleSlotConstantRefsIdentical(J9ROMConstantPoolItem * romCP1, U_32 index1,
 	return utfsAreIdentical(J9ROMSTRINGREF_UTF8DATA((J9ROMStringRef *) ref1), J9ROMSTRINGREF_UTF8DATA((J9ROMStringRef *) ref2));
 }
 
+/*
+ * Helper methods for areCallSiteDataMethodsEquivalent
+ */
+
+/* ROM call site data header size (16 bits of method handle address + additional argument count) */
+static const U_16 callSiteHeaderOffset = 2;
```

Comment:
This might be better as a define (I suspect this will create an actual variable in memory).